airflow/models/baseoperator.py

```diff
@@ -566,7 +566,20 @@ def priority_weight_total(self):
     @cached_property
     def operator_extra_link_dict(self):
         """Returns dictionary of all extra links for the operator"""
-        return {link.name: link for link in self.operator_extra_links}
+        from airflow.plugins_manager import operator_extra_links
+
+        op_extra_links_from_plugin = {}
+        for ope in operator_extra_links:
```

Comment:
```suggestion
        for ope in self.operator_extra_links:
```

If this doesn't fail tests our tests need improving!

Reply:
no, it is intentional.

`operator_extra_links` is from plugins

`self.operator_extra_links` is from the operator itself

Reply:
Check line 569

Reply:
Temporary blindness!